setup.py

```diff
@@ -17,7 +17,7 @@ def read_requirements(file):
 setup(
     name="torchtune",
     version="0.0.1",
-    packages=find_packages(),
+    packages=find_packages(exclude=["tests", "tests.*", "recipes"]),
```

Comment:
I take it excluding "recipes.*" was not needed?

Reply:
That's correct, `tests.*` is needed because there are `__init__.py` files within all sub-directories of `tests/`, e.g. `tests/torchtune/generation/__init__.py`, etc.

But there is only one `__init__.py` file in `recipes/`, so `recipes.*` is not needed.